packages/compass-databases-navigation/src/icon-button.tsx

```diff
@@ -6,22 +6,46 @@ import {
   spacing,
   css,
   cx,
-  uiColors,
 } from '@mongodb-js/compass-components';
+import {
+  darkIconColor,
+  darkIconColorHover,
+  lightIconColor,
+  lightIconColorHover,
+} from './constants';
 
 const iconContainer = css({
-  color: uiColors.white,
   display: 'block',
   flex: 'none',
   fontSize: 0,
 });
 
+export type IconMode = 'normal' | 'hovered';
```

Comment:
Trying to differentiate between the items that go straight on the sidebar background and area always visible like the expand/collapse one and the database or collection items vs the ones that only show up when you hover over the tree item (like the ... next to collections and add or delete next to databases).

This is because in the designs they have different colours.

So far these are the best names I could come up with. Maybe variant rather than mode? Suggestions welcome and then I can find/replace them.

Reply:
My first instinct would be to call this IconState, but both mode and variant sounds good to me 👍

Reply:
I think that mode fits pretty well here.